hangups/ui/__main__.py

```diff
@@ -165,7 +163,8 @@ def _on_connect(self):
         self._urwid_loop.widget = self._tabbed_window
 
     def _on_event(self, conv_event):
-        """Open conversation tab for new messages when they arrive."""
+        """Open conversation tab for new messages when they arrive.
+           Pass conv events to notifier, if not diabled."""
```

Comment:
`diabled` -> `disabled`

Also, for multi-line docstrings, there should be a blank line after the summary line.